src/main/java/io/pivotal/cla/data/repository/CorporateSignatureRepository.java

```diff
@@ -32,7 +32,7 @@
 
 	default CorporateSignature findSignature(String claName, Collection<String> organizations, Collection<String> emails) {
 		PageRequest pageable = new PageRequest(0, 1);
-		List<String> emailDomains = emails == null ? Collections.emptyList() : emails.stream().map( e-> e.substring(e.lastIndexOf("@") + 1)).collect(Collectors.toList());
+		List<String> emailDomains = emails == null ? Collections.singletonList("") : emails.stream().map( e-> e.substring(e.lastIndexOf("@") + 1)).collect(Collectors.toList());
```

Comment:
We should add `NOT NULL` to the `corporate_signature.email` column to make sure we get the right results. Wasn't quite sure about the `NOT NULL` constraints.